examples/gno.land/r/demo/grc20factory/grc20factory.gno

```diff
@@ -23,56 +30,68 @@ func NewWithAdmin(name, symbol string, decimals uint, initialMint, faucet uint64
 		panic("token already exists")
 	}
```

Comment:
I like the idea of using an ID instead of a symbol. 👍 

Regarding "the real USDT," I believe that responsibility lies not with this contract but with a DAO-powered token curation contract, perhaps `grc20vanity`.